rclpy/rclpy/exceptions.py

```diff
@@ -109,7 +109,7 @@ class InvalidParameterTypeException(ParameterException):
 
     from rclpy.parameter import Parameter
 
-    def __init__(self, desired_parameter: Parameter, expected_type: Parameter.Type) -> None:
+    def __init__(self, desired_parameter: Parameter, expected_type: str) -> None:
```

Comment:
Yes when I initially did the type for that exception I mistakenly thought it took the Parameter.Type object rather than the name field of that object.